api/api.ts

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+const api = axios.create({
+  baseURL: '/api/',
```

Comment:
In case our server doesn't respond

```suggestion
  baseURL: '/api/',
  timeout: 10 * 1000, // 10 seconds.
```